### PR TITLE
feat: add warning if not part of the nearform-actions org

### LINKED ISSUE #114: Create warning if not part of the new nearform-actions organisation

### PR DESCRIPTION
**Pull request by gregoryduckworth**
Closes #114

### CODE REVIEW

**Review by simoneb — CHANGES_REQUESTED** [review]
I don't think this is correct, and the existing warning is also incorrect because it uses the GITHUB_REPOSITORY env var which contains the name of the **using** repository, not that of the action. The existing warning also needs to be fixed. Please see the env vars available here https://docs.github.com/en/actions/learn-github-actions/variables

**Comment by gregoryduckworth** [comment]
Ok, had a go as I understand it with this. Do we have anywhere that we can test this action?

File: src/index.js [hunk 1]
```diff
@@ -23,6 +23,21 @@ function logActionRefWarning() {
...
+/**
+ * Displays warning message if the repository is under the nearform organisation
+ */
+function logRepoWarning() {
+  const repoName = process.env.GITHUB_ACTION_REPOSITORY
+
+  if (repoName.split('/')[0] != 'nearform-actions') {
+    core.warning(
+      `${repoName} is not under the nearform-actions organisation` +
+        `Please update the repository to be 'nearform-actions'`
```
Comment (simoneb) [inline src/index.js:35]: I think this should be worded differently. When we merge this, dependabot will bump this package in all actions (one thing that we could do is check that indeed it is being used by all our actions, which I'm not 100% sure of). When those actions are executed, if the workflows reference the action from the nearform org, they will get this message.

So this message should reflect that the old org is no longer valid and that the user should update their workflows. I would basically reflect this better in the warning message, especially considering that this message will start appearing _before_ actions are actually moved.

**Comment by gregoryduckworth** [comment]
> No easy way to test this, but I'm confident this works correctly

Just looking into this a little more, is it worth moving the new `logRepoWarning` into the `logActionRefWarning` [see](https://github.com/nearform/github-action-notify-release/blob/main/src/index.js#L8)? Otherwise we will have to update all the repositories and we potentially might miss one as the function is never called?

**Pull request merged**